crates/astria-sequencer-relayer/tests/it/data_availability.rs

```diff
@@ -131,30 +81,8 @@ async fn celestia_client() {
         .submit_block(block, &signing_key, verification_key)
         .await
         .unwrap();
-    let height = submit_block_resp
+    let _height = submit_block_resp
         .namespace_to_block_num
-        .get(&DEFAULT_NAMESPACE.to_string())
+        .get(&DEFAULT_NAMESPACE)
```

Comment:
should there also be a unit test to test fetching + deserializing the `SignedNamespaceData`?

Reply:
Do you mean putting it on Celestia and then fetching it from there?